includes/model/class-user.php

```diff
@@ -233,9 +233,44 @@ public function get_endpoints() {
 	 * @return array The extended User-Output.
 	 */
 	public function get_attachment() {
-		$array = array();
+		$extra_fields = new WP_Query(
+			array(
+				'post_type'      => 'ap_extrafield',
+				'posts_per_page' => -1,
```

Comment:
You can use `'nopaging' => true` in place of -1 on `posts_per_page` for a bit more clarity.